core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -361,6 +361,16 @@ object Schema extends LowPrioritySchema with SchemaCompanionMacros {
     val Attribute: AttributeKey[Tuple] = new AttributeKey[Tuple]("sttp.tapir.Schema.Tuple")
   }
 
+  /** For coproduct schemas, when there's a discriminator field, used to attach the encoded value of the discriminator field. Such value is
```

Comment:
Coproduct may be also rendererd without a discriminator field (Json object nesting). We need to make sure this case is covered as well.

Reply:
Yes, this applies only to discriminators. I think if there's no discriminator, then we don't have to do anything, that is we shouldn't add any constraints?